src/components/RoundImageRow/RoundImageRow.js

```diff
@@ -1,11 +1,12 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import RoundImage from 'components/RoundImage'
+import cx from 'classnames'
 import './RoundImageRow.scss'
 
 const { array, string, bool } = PropTypes
 
-export default function RoundImageRow ({ imageUrls = [], inline = false, className, vertical, cap, ascending, ...rest }) {
+export default function RoundImageRow ({ imageUrls = [], inline = false, className, vertical, cap, ascending, count, blue, ...rest }) {
```

Comment:
I find it more readable and orderly seeming to go vertical when the props restructuring goes this long.